.github/workflows/release_stackablectl.yml

```diff
@@ -48,17 +44,6 @@ jobs:
       - name: Build for non-Windows
         if: matrix.os != 'windows-latest'
         run: cargo build --target ${{ matrix.target }} --release -p stackablectl
-      - name: Prepare Windows env
```

Comment:
For me this are valuable information once we try to get the Windows build to work.
You sure, you can get it from the git history, but you need to know *that* and *where* to look.

Reply:
Yeah I want to eventually replace this with the working build instructions for Windows.

Reply:
We will continue to not include the Windows build. I will also remove this commented out stuff, as it is not even close to what the final solution will look like (from my own experiments).

Reply:
Alright!